app/lib/features/chat_ng/widgets/events/replied_to_event.dart

```diff
@@ -89,30 +89,30 @@ class OriginalEventItem extends ConsumerWidget {
     final eventType = item.eventType();
     return switch (eventType) {
       // handle message inner types separately
-      'm.room.message' => buildReplyMsgEventItem(
-        roomId,
-        messageId,
-        item,
-        isUser,
-      ),
+      'm.room.message' => buildReplyMsgEventItem(context, roomId, item, isUser),
       'm.room.redaction' => RedactedMessageWidget(),
       'm.room.encrypted' => EncryptedMessageWidget(),
       _ => _buildUnsupportedMessage(eventType),
     };
   }
 
   Widget buildReplyMsgEventItem(
+    BuildContext context,
     String roomId,
-    String messageId,
     TimelineEventItem item,
     bool isUser,
   ) {
     final msgType = item.msgType();
     final content = item.msgContent();
+    final messageId = item.eventId();
```

Comment:
we had to change the logic a bit, as the `originalMessageId` is now referring to the event we are _within_, and the widgets expect us to provide the event they are actually rendering.